test/utils/session-helper.test.js

```diff
@@ -87,3 +90,57 @@ describe('setDocAclForUser', () => {
     expect(resourceAcl).toEqual(mockResourceAcl);
   });
 });
+
+describe('getContainerUrl', () => {
```

Comment:
These tests are good.

Reply:
Thanks, I'll see what I can do about the tests in session-core.